sdk/trace/trace_test.go

```diff
@@ -208,6 +208,28 @@ func TestSampling(t *testing.T) {
 	}
 }
 
+func TestStartSpanWithExplicitSpanContext(t *testing.T) {
```

Comment:
Please add test for invalid explicit span context.

Reply:
Done!